deel/torchlip/modules/module.py

```diff
@@ -44,6 +44,25 @@
 logger = logging.getLogger("deel.torchlip")
 
 
+def vanilla_model(model: nn.Module):
```

Comment:
This function might be added to the `__all__` variable in `deel/torchlip/__init__.py`. It could then be accessed directly from `deel.torchlip` import.